roles/contrail-provision-dockers/templates/inventory/group_vars/all.yaml.j2

```diff
@@ -0,0 +1,16 @@
+---
+BUILD_VMS: false
+CONFIGURE_VMS: true
+CREATE_CONTAINERS: true
+CONTAINER_VM_CONFIG:
```

Comment:
This seems to be configuration related to spawning a VM for containers? This is going to be done by nodepool and should probably not be here - we are not going to spawn VMs inside of another VMs.

Reply:
Right first variable controls spawning VMs so I disabled related task setting variable to false. Second is temporary on until nodepool image will be provided with dependencies.

Reply:
What depenencies are those? Do you have a list somewhere?

Reply:
Yes, I have a list. It's a list matching following playbook: https://github.com/Juniper/contrail-ansible-deployer/blob/master/playbooks/roles/configure_container_hosts/tasks/install_software.yml